reset-dev-env.bash

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+npm install
+
+poetry install --extras='cdk datasets-endpoint'
```

Comment:
Why `poetry install` is executed before Python venv was activated ?

Reply:
That's how Poetry works - it bootstraps the virtualenv before installing the software.